tests/services/test_tokens.py

```diff
@@ -0,0 +1,144 @@
+from datetime import datetime, timedelta
+from unittest.mock import AsyncMock
+from uuid import UUID
+
+import pytest
+
+from auth_app.models import RefreshTokenORM
+from auth_app.models.users import (
+    UserORM,
+    UserRole,
+)
+from auth_app.schemes.tokens import RoleDataScheme
+from auth_app.schemes.users import (
+    AuthUserScheme,
+    GetUserScheme,
+    RoleEnum,
+)
+from auth_app.services.tokens import TokenService
+from auth_app.services.users import UserService
+from auth_app.services.utils.token_handler import TokenData
+
+auth_data = AuthUserScheme(
+    **{
+        "email": "example@email.com",
+        "password_hash": "examplePassword123!",
+    }
+)
```

Comment:
Тут вообще не понимаю в чём прикол? Зачем так объект класса создавать ??
```suggestion
auth_data = AuthUserScheme(
        email="example@email.com",
        password_hash="examplePassword123!",
)
```

Reply:
согласен, не вспомню уже причину такого оформления)